actions/TestTaker.php

```diff
@@ -163,6 +163,7 @@ public function editSubject()
         $myForm = $myFormContainer->getForm();
 
         $subjectUri = $subject->getUri();
+        $userService = tao_models_classes_UserService::singleton();
```

Comment:
Use of `singleton()` is deprecated, this service can be pulled from ServiceLocator. Easy to make it better than before: 
```
$this->getServiceLocator()->get(tao_models_classes_UserService::SERVICE_ID)
```

Reply:
done